azurerm/resource_arm_policy_definition.go

```diff
@@ -56,6 +58,11 @@ func resourceArmPolicyDefinition() *schema.Resource {
 				}, true),
 			},
 
+			"management_group_id": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
since it's not possible to migrate between policy definitions between management group's, I think this also wants:

```suggestion
				Optional: true,
				ForceNew: true,
```

Reply:
updated.